horace_core/sqw/@d4d/d4d.m

```diff
@@ -1,5 +1,5 @@
 classdef d4d < DnDBase
-    %D4D Create an 2-dimensional DnD object
+    %D4D Create an 3-dimensional DnD object
```

Comment:
seems to be 4D but you have corrected 2 to 3

Reply:
Indeed, this should be 4, will change it